gui/autotests/autotests.pro

```diff
@@ -0,0 +1,10 @@
+QT += testlib
+QT += gui
+CONFIG += qt warn_on depend_includepath testcase
+
+TEMPLATE = app
+
+QMAKE_CXXFLAGS += --coverage
+QMAKE_LFLAGS += --coverage
+
+SOURCES +=  tst_gui.cpp
```

Comment:
Will this be able to build multiple test binaries each with a main() function? E.g. a bunch of unit tests one per library + a few integration tests of the GUI itself?

Reply:
As far as I'm aware yes

Reply:
Could you demonstrate that by adding a second test file in the same directory producing a second test binary? Ideally, to add a new unit test, one would need to only author a new foo_test.cpp file, with no other changes to any scripts or configuration.